source/overview/license-and-subscription.rst

```diff
@@ -71,13 +71,31 @@ You may incur retroactive charges for any unique users added that exceed the lic
 Renewing an Enterprise license
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-The ability to renew your license in the Customer Portal will be available in a future release.
+You can renew your Mattermost license with a credit card if you have a standard Mattermost contract. When you renew your license, you can also increase the number of active users.
 
-To renew your license in the meantime, contact your Customer Success Manager to start the renewal process. Alternatively, `contact us <https://mattermost.com/contact-us/>`_ for assistance.
+If you are a reseller, have a non-standard contract, or want to adjust the number of active users on your license please contact sales@mattermost.com.
+
+System Admins will be alerted 60 days prior to the license expiry via a banner in Mattermost. Select **Renew license now** to start the renewal process in the Customer Portal. You can also dismiss the banner and renew your license at a later date via **System Console > Edition and License**.
+
+The renewal page in the Customer Portal lists your license information and contact details which is pre-populated based on your existing license subscription. 
+
+**To process your license renewal:**
+
+1. Enter your contact and billing information.
+2. Confirm the Mattermost Enterprise Edition version.
+3. Confirm the listed number of active users is correct.
+4. Select **Complete purchase**. 
+
+An email with the new license key and information on how to upload the license in the System Console will be sent to the primary email address.
```

Comment:
Question: who's primary email address? The System Admin's? (If a person can change the primary email address within Mattermost, it may be valuable to include location details).

Reply:
That's a good question - @jasonblais?

Reply:
It's the contact entered in the renewal form.

Reply:
To confirm; there's an account details field and then additional contact field. They're both pre-filled but also editable from what I can see, so they can enter any address and the next time they renew they can change that address should it have changed?

Reply:
Only the Account Details and the license information (edition + # of licensed seats) are pre-populated, and are based on the email associated with the original license purchase.

And yes, all fields can be edited as needed.